templates/repo/issue/view_content/comments.tmpl

```diff
@@ -182,5 +182,17 @@
 			</a>
 			<span class="text grey"><a href="{{.Poster.HomeLink}}">{{.Poster.Name}}</a> {{$.i18n.Tr "repo.issues.cancel_tracking_history"  $createdStr | Safe}}</span>
 		</div>
+	{{else if eq .Type 16}}
+		<div class="event">
+			<span class="octicon octicon-primitive-dot"></span>
+			<a class="ui avatar image" href="{{.Poster.HomeLink}}">
+				<img src="{{.Poster.RelAvatarLink}}">
+			</a>
+			<span class="text grey"><a href="{{AppSubUrl}}/{{.RepoFullName}}/commit/{{.CommitSHA}}">{{.Content | Str2html}}</a></span>
```

Comment:
Please create method for Comment type to generate target URL and use it here and lower